components/fieldgroups/medication/healthcare-medication.schema.json

```diff
@@ -73,11 +65,6 @@
               "type": "number",
               "description": "Amount of drug in the package."
             },
-            "xdm:volume": {
-              "title": "Drug Volume",
-              "type": "number",
-              "description": "Drug Volume."
-            },
             "xdm:ingredients": {
```

Comment:
Reminder to add form of medication- 
Description: form that the medication comes in such as tablet, capsule, or liquid